### PR TITLE
[REVIEW READY] Removes epithets/major fourth wall breaks/etc.

### PR DESCRIPTION
**Pull request by Shadeykins**
Kbai.

## About The Pull Request

Does exactly what it says on the tin.

## Changelog
:cl:
tweak: Changes the name/references of a food item to meatball.
tweak: Changes a bunch of the brain damage strings.
tweak: Clears out a lot of pre-inputted names in some config/.txt files that shouldn't be there.
/:cl:

### CODE REVIEW

**Comment by AffectedArc07** [comment]
DNM until other commit was added

**Comment by Shadeykins** [comment]
Also wrapped in some general brain damage changes in here too.

File: strings/ion_laws.json [hunk 1]
```diff
@@ -483,7 +453,6 @@
         "MUMBLE",
         "SPEAK IN HAIKU",
         "BE EFFICIENT",
-        "HAVE A PLAN TO KILL EVERYONE YOU MEET",
```
Comment (AffectedArc07) [inline strings/ion_laws.json]: RIP TF2 sniper ref

File: strings/names/death_commando.txt [hunk 2]
```diff
@@ -1,70 +0,0 @@
-A whole bunch of spiders in a SWAT suit
-Al "Otta" Gore
-AMERICA
-Beat Punchbeef
```
Comment (AffectedArc07) [inline strings/names/death_commando.txt]: Remove AMERICA and the george melons, we need atleast 1 DS name

**Comment by AffectedArc07** [comment]
Fuck it ill speedmerge a fix in

**Comment by Shadeykins** [comment]
rip

**Pull request merged**